vendor/engines/secure_rooms/db/migrate/20170406162317_create_events.rb

```diff
@@ -12,8 +12,13 @@ def change
       t.timestamps null: false
     end
 
-    add_index :secure_rooms_events, :user_id
-    add_index :secure_rooms_events, :card_reader_id
+    if NUCore::Database.oracle?
+      add_index :secure_rooms_events, :user_id, name: "index_rooms_events_on_user_id"
```

Comment:
If we're creating custom names for an index, we should use that name on both oracle and mysql to keep them consistent.